.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -1,32 +1,45 @@
 ---
-name: Bug report
+name: Bug Report
 about: Create a report to help us improve
 title: ''
 labels: ''
 assignees: ''
 
 ---
 
-# Please follow the general troubleshooting steps first:
+Note: If the bug affects multiple games, please open an issue for each game with the game in the title.
```

Comment:
```suggestion
<!-- Note: If the bug affects multiple games, please open an issue for each game with the game in the title. -->
```

Reply:
Secondly I'm not sure this policy actually makes sense, if there's an ALVR issue that triggers problems in multiple games then that's just one issue for our POV.

Reply:
My primary goal here was fixing formatting, but I do agree that it seems like a silly policy. If you'd like to change it we can, I just assumed there was a reason it was there.

Reply:
Yeet it away~